og_ui/og_ui.admin.inc

```diff
@@ -998,9 +998,29 @@ function theme_og_ui_admin_permissions($variables) {
  * Allow site admin to add or remove group fields from fieldable entities.
  */
 function og_ui_field_settings($form, &$form_state) {
+
```

Comment:
As far as I know, we don't use three empty lines, two empty lines should be enough.